admin/common_auth/models.py

```diff
@@ -47,6 +47,10 @@ class MyUser(AbstractBaseUser, PermissionsMixin):
     date_joined = models.DateTimeField(default=datetime.now)
     confirmed = models.BooleanField(default=False)
     osf_id = models.CharField(default=False, max_length=10, blank=True)
+    desk_email = models.EmailField(verbose_name='desk email', max_length=255,
+                                   default='')
+    desk_password = models.CharField(verbose_name='desk password',
+                                     max_length=128, default='')  # TODO: encrypt
```

Comment:
The Desk PR is the furthest along, and was up-to-date as of yesterday, broken again...

As to the password, this basically needs to send the plaintext password to Desk.com, or I just can't find documentation as to what to send instead. There is the possibility of using Oauth, thoughts @mfraezz ?